sdk/containerregistry/container-registry/src/util.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Handles etag quoting in AppConfig response objects.
```

Comment:
AppConfig referenced.

Reply:
Going to remove this file. It's from template, not useful to this library.